webroot/results/includes/_database.php

```diff
@@ -166,7 +166,7 @@ function getAllUsedCountriesCompetitions () {
 #----------------------------------------------------------------------
 function getAllUsedContinents () {
 #----------------------------------------------------------------------
-  return dbQuery("SELECT * FROM Continents");
+  return dbQuery("SELECT * FROM Continents WHERE recordName <> ''");
```

Comment:
I looked at how we handle the fake countries, and actually what we do is to join on the `Records` table, which indeed returns only real countries.
I'm not sure what's going to be the performance loss of one additional query with an inner join vs this simple select, but that could be a way of selecting only the real continents.